src/stac_to_dc/adapters/repository.py

```diff
@@ -13,7 +13,10 @@ def __init__(self, s3: S3):
     def get_catalogs_from_path(self, bucket: str, path: str) -> List[dict]:
         catalog_objs = self.s3.list_objects(bucket, prefix=path, suffix='catalog.json')
         return [self.s3.get_json_object(c.bucket_name, c.key) for c in catalog_objs]
-
+    
```

Comment:
This says it returns a list but only decodes a single object. compared to the one above it seems to imply that every "file" is a single object.

Reply:
ah my bad didn't mean to put list, it returns a single object